clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/CommonServerStoreProxy.java

```diff
@@ -73,15 +73,14 @@
       long key = response.getKey();
       Chain evictedChain = response.getEvictedChain();
       LOGGER.debug("CLIENT: on cache {}, server requesting hash {} to be invalidated (evicted chain : {})", cacheId, key, evictedChain);
-      invalidation.onInvalidateHash(key, evictedChain);
+      invalidation.onEvictInvalidateHash(key, evictedChain);
```

Comment:
the above log message be reworded a tad to make it more explicit that an eviction happened?  (customer has been asking about enabling logging for those).

Reply:
I mean I guess we could... I'm more worried about the implication of turning on this logging).